test/ControlTests/ShotControllerTest.cpp

```diff
@@ -180,82 +149,13 @@ TEST(ShotControllerTest, getshotdata_test) {
     EXPECT_TRUE(shotdata.kicker);
     EXPECT_FALSE(shotdata.chipper);
     EXPECT_FLOAT_EQ(shotdata.angle, (world::world->getBall()->pos - robotWithBall->pos).toAngle());
-    EXPECT_FLOAT_EQ(shotdata.geneva, 3);
 
     // chip test
     shotdata = shotController.getRobotCommand(*robotWithBall, simulatedShotTarget, true, BallSpeed::MAX_SPEED, false, ShotPrecision::LOW);
     EXPECT_FALSE(shotdata.kicker);
     EXPECT_TRUE(shotdata.chipper);
     EXPECT_FLOAT_EQ(shotdata.angle, (world::world->getBall()->pos - robotWithBall->pos).toAngle());
-    EXPECT_FLOAT_EQ(shotdata.geneva, 3);
-
-    /*
-     *
-     * When the autogeneva is on we should test some things a lot -> in a loop
-     * when we chip the geneva from the outgoing command should ALWAYS be 3
-     * We alternate the target position to see the difference in behaviour
-     */
-    for (int i = 0; i < 100; i++) {
-        auto world = testhelpers::WorldHelper::getWorldMsg(1, 0, true, field);
-        rtt::world::world->updateWorld(world);
-        auto robot = world::world->getUs().at(0);
-        robot->setGenevaState(3);
-        robot->setWorkingGeneva(true);
-        robot->setTimeToChangeOneGenevaState(-1);
-
-        shotdata = shotController.getRobotCommand(*robot, testhelpers::WorldHelper::getRandomFieldPosition(field), true, BallSpeed::MAX_SPEED, true, ShotPrecision::HIGH);
-        EXPECT_FALSE(shotdata.kicker);
-        EXPECT_FLOAT_EQ(shotdata.geneva, 3);
-    }
```

Comment:
This tests more than just the geneva. Have you tested if ShotController still works without the geneva? I don't like these tests being removed.

Reply:
Same goes for all other tests in this file.

Reply:
I'm not 100% sure but I don't think the shotcontroller is used anymore. @Ratoone can you confirm this?

Reply:
The only occurrences of "ShotController" anywhere is in the file ShotControllerTest.cpp, and that file does not belong to any target.

Reply:
Ah ok. Then IMO we remove the entire class aswell, as it's functionality relied on geneva a lot on the past.

Reply:
Can we merge this first?

Reply:
Why not remove the class? I'll even do it for you if you don't want to. Keeping code that is unused, has no use and which tests have been removed makes no sense.

Reply:
If you could do it, that'd be great! I have no idea about the structure of the code anymore